### ISSUE TITLE
Scanning photos  with occ always crashes

### ISSUE THREAD

**Issue by Twilek-de** [nextcloud/maps#114]
Running latest NC beta on Apache PHP 7.3. Scanning photos always crashes at the same place after successfully scanning several photos with

```
In PelDataWindow.php line 241:

  [lsolesen\pel\PelDataWindowOffsetException]
  Offset 1556 not within [0, 1530]


Exception trace:
 () at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelDataWindow.php:241
 lsolesen\pel\PelDataWindow->validateOffset() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelDataWindow.php:366
 lsolesen\pel\PelDataWindow->getShort() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelIfd.php:183
 lsolesen\pel\PelIfd->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelIfd.php:221
 lsolesen\pel\PelIfd->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelIfd.php:221
 lsolesen\pel\PelIfd->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelTiff.php:159
 lsolesen\pel\PelTiff->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelExif.php:108
 lsolesen\pel\PelExif->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelJpeg.php:216
 lsolesen\pel\PelJpeg->load() at /var/www/nextcloud/apps/maps/vendor/lsolesen/pel/src/PelJpeg.php:126
 lsolesen\pel\PelJpeg->__construct() at /var/www/nextcloud/apps/maps/lib/Service/PhotofilesService.php:439
 OCA\Maps\Service\PhotofilesService->getExifPelBackup() at /var/www/nextcloud/apps/maps/lib/Service/PhotofilesService.php:385
 OCA\Maps\Service\PhotofilesService->getExif() at /var/www/nextcloud/apps/maps/lib/Service/PhotofilesService.php:269
 OCA\Maps\Service\PhotofilesService->addPhoto() at /var/www/nextcloud/apps/maps/lib/Service/PhotofilesService.php:64
 OCA\Maps\Service\PhotofilesService->rescan() at /var/www/nextcloud/apps/maps/lib/Command/RescanPhotos.php:77
 OCA\Maps\Command\RescanPhotos->rescanUserPhotos() at /var/www/nextcloud/apps/maps/lib/Command/RescanPhotos.php:68
 OCA\Maps\Command\RescanPhotos->execute() at /var/www/nextcloud/3rdparty/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:908
 Symfony\Component\Console\Application->doRunCommand() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:269
 Symfony\Component\Console\Application->doRun() at /var/www/nextcloud/3rdparty/symfony/console/Application.php:145
 Symfony\Component\Console\Application->run() at /var/www/nextcloud/lib/private/Console/Application.php:213
 OC\Console\Application->run() at /var/www/nextcloud/console.php:97
 require_once() at /var/www/nextcloud/occ:11
```

I have another NC Instance on a webhosting where Maps crashes as well scanning the photos.

**Comment by buoncri** [comment]
Having the same problem i found that a jpg file with zero byte crash the scan.

**Comment by tacruc** [comment]
I have similar issues. We will have tow fixes in the next release.

**Comment by Twilek-de** [comment]
Super I was just going to say that the file it is crashing on seems to be quite normal.

**Comment by tacruc** [comment]
I'm not an experd on jpg files, but my assumption is that something in the metadata of the file is wrong or non standard, such that the images is show correctly, but some implementation fail to decode the metadata.

**Comment by jancborchardt** [comment]
This will be fixed with #91 and the 0.1.2 release, right @tacruc?

**Comment by tacruc** [comment]
Should be fixed in 0.1.2
